src/main/java/com/datasonnet/CSVFormatPlugin.java

```diff
@@ -32,7 +32,7 @@ public Value read(String input, Map<String, Object> params) throws IOException {
         ObjectMapper mapper = new ObjectMapper();
         CsvSchema.Builder builder = this.getBuilder(params);
 
-        boolean useHeader = params != null && params.get(USE_HEADER) != null ? (Boolean)params.get(USE_HEADER) : true;
+        boolean useHeader = params != null && params.get(USE_HEADER) != null ? new Boolean(params.get(USE_HEADER).toString()) : true;
```

Comment:
It'd be good to make some utility functions designed for general use for this sort of "it might be an actual value or it might be a string" unpacking.

Reply:
We should also document, for plugin authors (I don't think we have the docs for them yet, so this is more future-looking) what they can expect params to come in like--for example, that primitives might be their actual types or might be strings of them